src/fparser/two/tests/fortran2008/test_submodule_stmt_r1117.py

```diff
@@ -92,44 +93,32 @@ def test_simple_error6():
 
 def test_simple_error7(monkeypatch):
     """Test the parsing of a submodule statement when there is a single
-    right hand bracket. The error generated here is unreachable if
-    splitparen works correctly so we need to monkeypatch.
+    right hand bracket.
 
     """
-
-    monkeypatch.setattr(
-        "fparser.common.splitline.splitparen", lambda x: ["", "id)", "name"]
-    )
+    monkeypatch.setattr(splitline, "splitparen", lambda x: ["", "id)", "name"])
     with pytest.raises(NoMatchError) as excinfo:
         dummy_ = Submodule_Stmt("submodule id) name")
     assert "Submodule_Stmt: 'submodule id) name'" in str(excinfo.value)
 
 
 def test_simple_error8(monkeypatch):
     """Test the parsing of a submodule statement when there is a single
-    left hand bracket. The error generated here is unreachable if
-    splitparen works correctly so we need to monkeypatch.
+    left hand bracket.
 
     """
-
-    monkeypatch.setattr(
-        "fparser.common.splitline.splitparen", lambda x: ["", "(id", "name"]
-    )
+    monkeypatch.setattr(splitline, "splitparen", lambda x: ["", "(id", "name"])
     with pytest.raises(NoMatchError) as excinfo:
         dummy_ = Submodule_Stmt("submodule (id name")
     assert "Submodule_Stmt: 'submodule (id name'" in str(excinfo.value)
 
 
 def test_splitparen_error(monkeypatch):
     """Test that if the first argument of the splitparen is not empty then
-    an error is returned.
+    an error is returned. Monkeypatch to force this error.
```

Comment:
This is all a bit worrying. Is it possible to fix splitparen instead?

Reply:
This is no different to the tests that were there before splitting out the code, other than a bit of reformatting. I suggest that fixing this issue - if needed - should be done in a separate PR.

Reply:
OK :-)